lms/static/js/spec/ccx/schedule_spec.js

```diff
@@ -0,0 +1,193 @@
+define(['js/ccx/schedule'],
+    function() {
+        describe("edx.ccx.schedule.ScheduleView", function() {
+            var view = null,
+                ajaxSuccess = true;
+
+            var assertAjax = function(url, method, data) {
+                expect($.ajax).toHaveBeenCalled();
+                var ajaxArgs = $.ajax.mostRecentCall.args[0];
+                expect(ajaxArgs.url).toEqual(url);
+                expect(ajaxArgs.type).toEqual(method);
+                expect(ajaxArgs.data).toEqual(JSON.stringify(data))
+            };
+
+            beforeEach(function() {
+                loadFixtures("js/fixtures/ccx/schedule.html");
+
+                var scheduleFixture = readFixtures("templates/ccx/schedule.underscore");
+                appendSetFixtures("<div id=\"schedule_template\">" + scheduleFixture + "</div>");
+		schedule_template = _.template($('#schedule_template').html())
+ 
+                save_url = 'save_ccx';
+
+                $.fn.leanModal = function(param) {
+		    return true;
+                }
+
+                // Stub AJAX calls to return success / failure
+                spyOn($, "ajax").andCallFake(function(e) {
+                    if (!ajaxSuccess) {
+                        e.error({});
+                    }
+                });
+
+                // Stub location.reload() to prevent test suite from reloading repeatedly
+                spyOn(edx.ccx.schedule, "reloadPage").andCallFake(function() {
+                    return true;
+                });
+
+                data = [{
+                    "category": "chapter",
+                    "display_name": "Introduction",
+                    "due": null,
+                    "start": null,
+                    "location": "i4x://edX/DemoX/chapter/d8a6192ade314473a78242dfeedfbf5b",
+                    "hidden": true,
+                    "children": [
+                    {
+                    "category": "sequential",
+                    "display_name": "Demo Course Overview",
+                    "due": null,
+                    "start": null,
+                    "location": "i4x://edX/DemoX/sequential/edx_introduction",
+                    "hidden": true,
+                    "children": [
+                    {"category": "vertical",
+                    "display_name": "Introduction: Video and Sequences",
+                    "due": null,
+                    "start": null,
+                    "location": "i4x://edX/DemoX/vertical/vertical_0270f6de40fc",
+                    "hidden": true}]}]
+                }];
+                view = new edx.ccx.schedule.ScheduleView({el: $('#new-ccx-schedule')});
+		view.schedule_collection.set(data)
+		view.render();
+
+
+            });
+
+            it("verifies correct view setup", function() {
+		expect(view.dirty).toBe(false);
```

Comment:
Indentation looks off from next line. There appear to be more throughout this file (L83, L90, etc) and I'm going to stop pointing them out. Could you do a pass and make sure all indentation is consistent (and not, perhaps, mixing tabs and spaces)?